.github/workflows/release.yaml

```
@@ -207,3 +207,12 @@ jobs:
         github_token: ${{secrets.GITHUB_TOKEN}}
         publish_branch: gh-pages
         publish_dir: www
+
+  container:
+    uses: ./.github/workflows/container.yaml
+    needs:
+      - checksum
+      - prerelease  
```

This is incorrect.
- You're not dependent upon the `checksum` job (_which uploads checksums after `package` job completes successfully_), but you are dependent upon `package` completing.
- Likewise for `prerelease` job, except images should probably only be for proper releases, so this would be fine if you added a conditional guard on this jobs output.

```suggestion
    needs:
      - package
      - prerelease
    # Avoid publishing images for pre-releases (NOTE: `value` is a string, not a boolean)
    if: ${{ needs.prerelease.outputs.value == 'false' }}
```